source/mainnet/net/guides/run-node-ubuntu.rst

```diff
@@ -121,6 +121,9 @@ Configure the node with baker keys
 
    Where you replace the path `/home/user/concordium/baker-credentials.json` with the actual location of the file.
 
+.. Note::
+   The path `%S/concordium-9dd9ca4d19e9393877d2c44b70f89acbfc0883c2243e5eeaecc0d1cd0503f478/` is the default path to the baker's state directory, where `9dd9ca4d19e9393877d2c44b70f89acbfc0883c2243e5eeaecc0d1cd0503f478` is the genesis hash.
```

Comment:
Could you also add this note to the testnet version of the documentation (changing the hash appropriately of course).

Reply:
It actually seems the same hash is used in the testnet documentation!? Is this a bug in the testnet document or did I mix up the documents?

Reply:
This is a bug in the documentation, the testnet genesis is, of course, 
`b6078154d6717e909ce0da4a45a25151b592824f31624b755900a74429e3073d`

So this PR is two for one :)

Reply:
Should be fixed.